homeassistant/components/onewire/switch.py

```diff
@@ -84,6 +84,20 @@ class OneWireSwitchEntityDescription(OneWireEntityDescription, SwitchEntityDescr
             for id in DEVICE_KEYS_0_7
         ]
     ),
+    "3A": (
+        OneWireSwitchEntityDescription(
+            key="PIO.A",
+            entity_registry_enabled_default=False,
+            name="PIO A",
+            read_mode=READ_MODE_BOOL,
+        ),
+        OneWireSwitchEntityDescription(
+            key="PIO.B",
+            entity_registry_enabled_default=False,
+            name="PIO B",
+            read_mode=READ_MODE_BOOL,
+        ),
+    ),
```

Comment:
```suggestion
    "3A": tuple(
        OneWireSwitchEntityDescription(
            key=f"PIO.{id}",
            entity_registry_enabled_default=False,
            name=f"PIO {id}",
            read_mode=READ_MODE_BOOL,
        )
        for id in DEVICE_KEYS_A_B
    ),
```

Reply:
Thanks -- It seems I made those changes on commit [c65395fb](https://github.com/home-assistant/core/pull/55921/commits/c65395fbf5921f89845975fc66f315d944435091). Are you fine with those?

Reply:
Yes, you can mark both comments as Resolved (I don't have rights to do that).